echopype/tests/echodata/test_echodata.py

```diff
@@ -15,15 +15,15 @@
 @pytest.fixture(scope="module")
 def single_ek60_zarr(test_path):
     return (
-        test_path['EK60'] / "ncei-wcsd" / "Summer2017-D20170615-T190214.zarr"
+        test_path['EK60'] / "ncei-wcsd" / "Summer2017-D20170615-T190214__NEW.zarr"
     )
 
 
 @pytest.fixture(
     params=[
         single_ek60_zarr,
         (str, "ncei-wcsd", "Summer2017-D20170615-T190214.zarr"),
-        (None, "ncei-wcsd", "Summer2017-D20170615-T190214.nc"),
+        (None, "ncei-wcsd", "Summer2017-D20170615-T190214__NEW.nc"),
```

Comment:
`Summer2017-D20170615-T190214.raw` converted using the code in this PR, and therefore with the beam group in `Sonar/Beam_group1` rather than `Beam`. See a Slack thread with @lsetiawan from 3/4 on the `echopype` channel.